redesign/portfolio.html

```diff
@@ -0,0 +1,139 @@
+          exclusive social network.</p>
+          <img src="./img/common/ellen-macarthur-foundation-logo.png" alt="Ellen MacArthur Foundation logo" class="w5 pt4"/>
+        </div>
+      </div>
+    </section>
+
+    <section class="dwyl-bg-mint center pt4-ns pb5-ns">
+
+      <div class="dib-ns w-50-ns dtc v-top">
+        <div class="w-80 w-60-ns white center tr">
+          <h1 class="mt0-ns mb3 pl4-ns f-xl fw9">Let's</h1>
+          <h1 class="pl4-ns mt3 f-xl fw9">Talk</h1>
+        </div>
+        <div class="w-80-ns w-90 ml3">
+          <a href="tel:+447708796446" class="link"><h2 class="tr white"><span class="dwyl-dark-gray fw9">Tel: </span>+44 (0) 7708 796446</h2></a>
+          <a href="mailto:hello@dwyl.io" class="link"><h2 class="tr white"><span class="dwyl-dark-gray fw9">Email:</span> hello@dwyl.io</h2><a/>
+          <h2 class="tr white"><span class="dwyl-dark-gray lh-copy fw9">Address:</span> 16 Palmers Road,<br>London, E2 0SY, UK</h2>
+          <p class="tr dwyl-dark-gray">(We'll get back to you within 24 hours)</p>
+        </div>
+      </div>
+
+      <div class="dib-ns bg-white pt4 pa2 shadow-4 ml5-ns w-40-ns">
+
+          <form id="gform" method="POST" class="tc tl-ns" action="https://script.google.com/macros/s/AKfycby5ZdD8HfCaz1tMVLw26qYQKEXkP8qdfwLek3TH_RYH3l1o1-Li/exec">
+          <!-- change the form action to your script url -->
+
+          <fieldset class="bn">
+            <label for="name" class="db mb2 tl pl3 pl0-ns">Your Name</label>
+            <input id="name" name="name" class="br2 ba dwyl-b--dark-gray pa2 w-90 border-box" placeholder="Tim Berners-Lee" />
+          </fieldset>
+
+          <fieldset class="bn">
+            <label for="email" class="db mb2 tl pl3 pl0-ns">Email Address</label>
+            <input id="email" name="email" class="br2 ba dwyl-b--dark-gray pa2 w-90 border-box" type="email" value="" required placeholder="tim@cern.com" />
+          </fieldset>
+
+          <fieldset class="bn">
+            <label for="message" class="db mb2 tl pl3 pl0-ns">Message</label>
+            <textarea id="message" name="message" class="br2 ba w-90 dwyl-b--dark-gray pa2" rows="10" placeholder="What problem can we solve for you?"></textarea>
+          </fieldset>
+
+          <button class="bn dib link mt2 mb3 ph3 pv3 dwyl-bg-orange white br2 f4 shadow-4 pointer montserrat">
+            <i class="fa fa-paper-plane"></i>&nbsp; Say Hello</button>
+        </form>
+
+        <div style="display:none;" id="thankyou_message">
+          <h2>
+            <em>Thanks</em> for contacting us! We will get back to you soon!</h2>
+        </div>
+
+        <!-- Submit the Form to Google Using "AJAX" -->
+        <script data-cfasync="false" type="text/javascript" src="https://cdn.rawgit.com/dwyl/html-form-send-email-via-google-script-without-server/master/form-submission-handler.js"></script>
+        <!-- END -->
+      </div>
+    </section>
+    <footer class="dwyl-bg-dark-gray white pt1 pt3-ns pb4 tc center">
```

Comment:
is it possible to `include` the footer dynamically rather than duplicating? 🤔 
(_it's fine for now as this is static ..._)